"npm run build:prod" issue.

Hello.

Thank you for your work.

I was following the installation process on the repo's main page, on `master` branch.
I tried to execute this step: `npm run build:prod` and got a SyntaxError.
Am I doing something wrong? 

I executed:
> alex@lptp:~/w/slamdata$ sudo npm run build:prod

I got:

> > slamdata@4.2.2 build:prod /home/alex/w/slamdata
> > npm run lint:crlf && npm run build:less && npm run build:icons && npm run build:psa && gulp bundle
> 
> 
> > slamdata@4.2.2 lint:crlf /home/alex/w/slamdata
> > gulp replace-crlf
> 
> /home/alex/w/slamdata/gulpfile.js:14
> const { injectIconsIntoHTML, createIconPureScript } = require("./script/icons");
>       ^
> 
> SyntaxError: Unexpected token {
>     at exports.runInThisContext (vm.js:53:16)
>     at Module._compile (module.js:374:25)
>     at Object.Module._extensions..js (module.js:417:10)
>     at Module.load (module.js:344:32)
>     at Function.Module._load (module.js:301:12)
>     at Module.require (module.js:354:17)
>     at require (internal/module.js:12:17)
>     at Liftoff.handleArguments (/home/alex/w/slamdata/node_modules/gulp/bin/gulp.js:116:3)
>     at Liftoff.<anonymous> (/home/alex/w/slamdata/node_modules/liftoff/index.js:198:16)
>     at module.exports (/home/alex/w/slamdata/node_modules/flagged-respawn/index.js:17:3)
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! slamdata@4.2.2 lint:crlf: `gulp replace-crlf`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the slamdata@4.2.2 lint:crlf script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /home/alex/.npm/_logs/2017-06-05T18_33_56_566Z-debug.log
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! slamdata@4.2.2 build:prod: `npm run lint:crlf && npm run build:less && npm run build:icons && npm run build:psa && gulp bundle`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the slamdata@4.2.2 build:prod script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /home/alex/.npm/_logs/2017-06-05T18_33_56_615Z-debug.log
>

--- a/README.md
+++ b/README.md
@@ -11,7 +11,7 @@ See the [Use with Quasar](#use-with-quasar) section for next steps.
 
 ## Building from source
 
-Building SlamData requires Node.js v4.1.0 or newer. The app is written in [PureScript](http://www.purescript.org/), the appropriate version of the PureScript compiler will be installed as part of the `npm` dependencies (see below).
+Building SlamData requires Node.js v6 or newer. The app is written in [PureScript](http://www.purescript.org/), the appropriate version of the PureScript compiler will be installed as part of the `npm` dependencies (see below).
 
 #### Prerequisites
 

